src/components/common/applicant-info-1.vue

```diff
@@ -347,7 +347,7 @@
                 :value="xproJurisdiction"
                 @blur="messages = {}"
                 @focus="handleFocus('xproJurisdiction', 'Business xproJurisdiction')"
-                @input="setNRData('xproJurisdiction', $event)"
```

Comment:
This didn't work after we reworked the actions. 
I had converted the applicant info and such, but must have missed the `nrData`. 

That object contains things like nature of business, additional info and xpro #'s etc.